tests/Makefile

```diff
@@ -19,79 +16,108 @@ all: minikube operator test
 # Start minikube
 .PHONY: minikube
 minikube:
-	minikube start --driver=docker
-	helm repo update
+	@echo "===== Starting minikube cluster. ====="
+	@minikube start --driver=docker
```

Comment:
Is there a reason for putting the `@` in front of all these Makefile commands?  My understanding is that this prevents the command from being printed to the output before executing it.  Is there a reason we don't want to see what commands are being run?

Reply:
To make the output cleaner. Some of the commands are super long now and were making it hard to read the output of the commands.